beacon-chain/rpc/apimiddleware/custom_hooks_test.go

```diff
@@ -732,6 +730,35 @@ func TestSerializeProduceV2Block(t *testing.T) {
 		require.NotNil(t, beaconBlock.Body)
 	})
 
+	t.Run("Bellatrix", func(t *testing.T) {
+		response := &produceBlockResponseV2Json{
+			Version: ethpbv2.Version_BELLATRIX.String(),
+			Data: &beaconBlockContainerV2Json{
+				BellatrixBlock: &beaconBlockBellatrixJson{
+					Slot:          "1",
+					ProposerIndex: "1",
+					ParentRoot:    "root",
+					StateRoot:     "root",
+					Body:          &beaconBlockBodyBellatrixJson{},
```

Comment:
Consider fill in execution_payload in `beaconBlockBodyBellatrixJson`. I like to have entropies for this kind of test